README.md

```diff
@@ -1,117 +1,84 @@
+install a stack, and provision cloud infrastructure using `kubectl`. Refer to
+the Crossplane [documentation] for comprehensive guides to using Crossplane,
+including deploying an application packaged as a stack.
 
-The full architecture and vision of the Crossplane project is described in depth in the [architecture document](https://docs.google.com/document/d/1whncqdUeU2cATGEJhHvzXWC9xdK29Er45NJeoemxebo/edit?usp=sharing). It is the best place to learn more about how Crossplane fits into the Kubernetes ecosystem, the intended use cases, and comparisons to existing projects.
+## Get Involved
 
-## Getting Started and Documentation
+* Discuss Crossplane on [Slack] or our [developer mailing list].
+* Follow us on [Twitter], or contact us via [Email].
+* Join our regular community meetings.
 
-For getting started guides, installation, deployment, and administration, see our [Documentation](https://crossplane.io/docs/latest).
-
-## Contributing
-
-Crossplane is a community driven project and we welcome contributions. See [Contributing](CONTRIBUTING.md) to get started.
-
-## Report a Bug
-
-For filing bugs, suggesting improvements, or requesting new features, please open an [issue](https://github.com/crossplaneio/crossplane/issues).
-
-## Contact
-
-Please use the following to reach members of the community:
-
-- Slack: Join our [slack channel](https://slack.crossplane.io)
-- Forums: [crossplane-dev](https://groups.google.com/forum/#!forum/crossplane-dev)
-- Twitter: [@crossplane_io](https://twitter.com/crossplane_io)
-- Email: [info@crossplane.io](mailto:info@crossplane.io)
-
-## Community Meeting
-
-A regular community meeting takes place every other [Tuesday at 9:00 AM PT (Pacific Time)](https://zoom.us/j/425148449).
-Convert to your [local timezone](http://www.thetimezoneconverter.com/?t=9:00&tz=PT%20%28Pacific%20Time%29).
-
-Any changes to the meeting schedule will be added to the [agenda doc](https://docs.google.com/document/d/1q_sp2jLQsDEOX7Yug6TPOv7Fwrys6EwcF5Itxjkno7Y/edit?usp=sharing) and posted to [Slack #announcements](https://crossplane.slack.com/messages/CEFQCGW1H/) and the [crossplane-dev mailing list](https://groups.google.com/forum/#!forum/crossplane-dev).
-
-Anyone who wants to discuss the direction of the project, design and implementation reviews, or general questions with the broader community is welcome and encouraged to join.
+The Crossplane community meeting takes place every other [Tuesday at 9:00am
+Pacific Time]. Anyone who wants to discuss the direction of the project, design
+and implementation reviews, or raise general questions with the broader
+community is encouraged to join.
 
 * Meeting link: https://zoom.us/j/425148449
-* [Current agenda and past meeting notes](https://docs.google.com/document/d/1q_sp2jLQsDEOX7Yug6TPOv7Fwrys6EwcF5Itxjkno7Y/edit?usp=sharing)
-* [Past meeting recordings](https://www.youtube.com/playlist?list=PL510POnNVaaYYYDSICFSNWFqNbx1EMr-M)
-
-## Project Status
```

Comment:
there is no mention of project status anymore that I see, do you think we should mention at least some summary of the status? and/or link to the API status table?

Reply:
I don't, to be honest.

My thinking is:

* This was useful to call out when we were a super early tech demo, but now that we're a little more mature (but not yet stable) I'm not sure it adds much value. I'd say we're approaching a similar level of maturity to most other projects in our space.
* The status of Crossplane as a project is increasingly split across repos; i.e. it's probably better to reflect the maturity of stack-aws over at stack-aws.
* All of our releases are 0.x.x and tagged as "pre-release" in Github, which I feel represents the status of Crossplane itself reasonably well.
* Anyone who decides to try out Crossplane will see that it's alpha when they get to the install steps.
* I surveyed a bunch of similar OSS project READMEs to inform this update, and found that it was rare for such details to be so front and center.
* The detailed status of each API group is still there in the docs